apps/shim-clr-ui/angular.json

```diff
@@ -37,7 +37,7 @@
               "node_modules/@cds/city/css/bundles/default.min.css",
               "node_modules/@clr/ui/clr-ui.min.css",
 
-              "node_modules/@cds/core/styles/module.tokens.min.css",
+              "node_modules/@cds/core/global.min.css",
```

Comment:
I thought `module.tokens.min.css` was enough. But I forgot the demos use `cds-layout`. So to make `cds-layout` work, I am loading the global style now.